yarn.lock

```diff
@@ -3153,9 +3153,9 @@ fast-safe-stringify@^2.0.8, fast-safe-stringify@^2.1.1:
   integrity sha512-W+KJc2dmILlPplD/H4K9l9LcAHAfPtP6BY84uVLXQ6Evcz9Lcg33Y2z1IVblT6xdY54PXYVHEv+0Wpq8Io6zkA==
 
 fast-xml-parser@^4.2.0:
-  version "4.4.1"
-  resolved "https://registry.yarnpkg.com/fast-xml-parser/-/fast-xml-parser-4.4.1.tgz#86dbf3f18edf8739326447bcaac31b4ae7f6514f"
-  integrity sha512-xkjOecfnKGkSsOwtZ5Pz7Us/T6mrbPQrq0nh+aCO5V9nk5NLWmasAHumTKjiPJPWANe+kAZ84Jc8ooJkzZ88Sw==
+  version "4.3.2"
```

Comment:
is has lately been upgraded to 4.4.1 (https://github.com/stoplightio/prism/pull/2570), something went wrong with the merge ?

Reply:
@lukaszzazulak  Hmm sorry unsure why this happened, I pulled the changes from the master branch and did a fresh reinstall of all packages based on the updated package.json file I received. I just copy-pasted the code from #2570 into the yarn.lock file and hopefully this resolves things!